service/src/test/java/bio/terra/cbas/dependencies/sam/TestSamService.java

```diff
@@ -176,13 +206,24 @@ void testGetSamUserExpiredToken() {
 
   @Test
   void testGetSamUserInterruptingToken() {
-    setTokenValue(tokenCausingInterrupt);
+    setTokenValue(tokenCausingUserInterrupt);
     SamInterruptedException e =
         assertThrows(SamInterruptedException.class, () -> samService.getSamUser());
     assertEquals("Request interrupted while getting user status info from Sam", e.getMessage());
     assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, e.getStatusCode());
   }
 
+  @Test
```

Comment:
If auth check configuration option is disabled, getSamUser will return an empty user, so endpoints will still be usable for local development without providing updated bearer tokens.